crypto/digestible/signature/Cargo.toml

```
@@ -0,0 +1,15 @@
+[package]
+name = "mc-crypto-digestible-signature"
+version = "1.0.0"
+authors = ["MobileCoin"]
+edition = "2018"
+description = "Semantic Signatures by MobileCoin"
```

Since the readme says "Signatures for Digestibles," should this say the following?
```suggestion
description = "Digestible Signatures by MobileCoin"
```